docs/developers/eks-demo.md

```diff
@@ -55,11 +55,11 @@ export CLUSTER_NAME=<eks-cluster-name>
 export AWS_REGION=<aws-region>
 ```
 
-3. Deploy the application. An example config template can be found [here](../../deployment-template/eks/eks-sidecar.yaml).
+3. Deploy the application. An example config template can be found [here](../../deployment-template/eks/otel-cloudwatch-deployment.yaml).
```

Comment:
should we keep `sidecar` keyword? eg `otel-cloudwatch-sidecar.yaml`

Reply:
Done